views/layouts/main.hbs

```diff
@@ -26,7 +25,8 @@
     {{> formatjs}}
     {{> syntax-highlighting}}
 
-    <script src="/js/app.js"></script>
+    {{section-for 'footer'}}
```

Comment:
Oh I see. Let's change the name to `'page-scripts'` or something. I was thinking "footer" as in content at the bottom of the page.